scripts/generate_pinval/generate_pinval.py

```diff
@@ -616,10 +614,8 @@ def main() -> None:
     # part of `ccao.vars_dict`
     key_map["combined_bldg_sf"] = "Combined Bldg. Sq. Ft."
 
-    PRESERVE = {"loc_latitude", "loc_longitude"}
-
     def pretty(k: str) -> str:
-        return k if k in PRESERVE else key_map.get(k, k)
```

Comment:
I don't 100% recall what our intention was here -- my best guess is that at some point the `PRESERVE` variable was consolidated with `NUMERIC_PRESERVE`, and we were using it in the same way to prevent bad number formatting -- but as far as I can tell all it's doing now is preventing `loc_latitude` and `loc_longitude` from getting a human-readable name in the chars table, so I cleaned it up.